docs/admin/install.md

```diff
@@ -0,0 +1,67 @@
+# Installing the App in Nautobot
+
+## Prerequisites
+
+- The plugin is compatible with Nautobot 1.0.3 and higher.
```

Comment:
Since `develop` is going to be 1.3.0 I think this needs to be updated as per the compat matrix.

Reply:
updated to specify Nautobot 1.4.0.